core/src/main/resources/org/apache/spark/ui/static/utils.js

```diff
@@ -161,7 +161,15 @@ function setDataTableDefaults() {
 
 function formatDate(date) {
   if (date <= 0) return "-";
-  else return date.split(".")[0].replace("T", " ");
+  else {
+     var d = new Date(date.replace("GMT", "Z"))
+     return d.getFullYear() + '-' +
```

Comment:
does this pad with zeroes properly - see formatTimeMillis?  From your screen shot it looks like it does but I'm wondering if that is browser specific.

Reply:
Done，due to `timeMillis` is `int` or`long` type and `formatDate` is string param, I extract a common method `formatDateString` to format `DateType` in js.